README.md

```diff
@@ -2,30 +2,53 @@
 
 ## Description
 
-A basic set of backup plans that can be consistantly created used accross the
+A basic set of backup plans that can be consistantly created used across the
 SBA organization.
 
+### Features
+
+* Configures the AWS Backup service to run weekly and quarterly
+* Weekly backups are retained for 90 days
+* Quarterly backups are retained forever
+* Out of the box will look for tag key-values of `BackupQuarterly: true` and `BackupWeekly: true` to initiate a backup
+* Grants necessary service role permissions
+
 ## Usage
 
+### Super Simple
 
-```
+```terraform
 module "backup-plans" {
   source  = "USSBA/backup-plans/aws"
-  version = "~> 1.0.2"
+  version = "~> 2.0"
```

Comment:
If the latest version is 3.0, let's point our examples to that

Reply:
done